### PR TITLE
chore: bump kommander ui

### PR DESCRIPTION
**Pull request by juliangieseke**

### CODE REVIEW

File: stable/kommander/requirements.lock [hunk 1]
```diff
@@ -4,7 +4,7 @@ dependencies:
   version: 0.1.4
 - name: kommander-cluster-lifecycle
   repository: https://mesosphere.github.io/kommander-cluster-lifecycle/charts
-  version: 0.3.4
+  version: v0.3.5
```
Comment (juliangieseke) [inline stable/kommander/requirements.lock]:
```suggestion
  version: 0.3.5
```

Comment (sebbrandt87) [reply stable/kommander/requirements.lock]: well that is what `helm dependency update` did :P 
and it is correct https://github.com/mesosphere/kommander-cluster-lifecycle/releases

**Comment by sebbrandt87** [comment]
Releases manual checked on a cluster. 
Working! 8)

**Comment by natmegs** [comment]
We tested and confirmed that we can provision aws clusters with both static credentials and roles-based credentials 👍

**Pull request merged**